drivers/volume/gce/gce.go

```diff
@@ -0,0 +1,45 @@
+package gce
+
+import (
+	torpedovolume "github.com/portworx/torpedo/drivers/volume"
+	"github.com/portworx/torpedo/drivers/volume/portworx/schedops"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	// DriverName is the name of the gce driver implementation
+	DriverName = "gce"
+	// GceStorage GCE storage driver name
+	GceStorage torpedovolume.StorageProvisionerType = "gce"
+)
+
+// Provisioners types of supported provisioners
+var provisioners = map[torpedovolume.StorageProvisionerType]torpedovolume.StorageProvisionerType{
+	GceStorage: "kubernetes.io/gce-pd",
+}
+
+type gce struct {
+	schedOps schedops.Driver
+	torpedovolume.DefaultDriver
+}
+
+func (d *gce) String() string {
+	return string(GceStorage)
+}
+
+func (d *gce) Init(sched string, nodeDriver string, token string, storageProvisioner string) error {
+	logrus.Infof("Using the GCE volume driver with provisioner %s under scheduler: %v", storageProvisioner, sched)
+	// Set provisioner for torpedo
+	if storageProvisioner != "" {
+		if p, ok := provisioners[torpedovolume.StorageProvisionerType(storageProvisioner)]; ok {
+			torpedovolume.StorageProvisioner = p
+		}
+	} else {
+		torpedovolume.StorageProvisioner = torpedovolume.DefaultStorageProvisioner
```

Comment:
This will end up using `portworx` as the provisioner. We should just return error instead

Reply:
Returning error here.